### PR TITLE
APIS-3739 - isTrial flag on ApiAccess

### PR DESCRIPTION
**Pull request by rose-garratt**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala [hunk 1]
```diff
@@ -256,5 +256,13 @@ class DocumentationServiceSpec extends UnitSpec with ScalaFutures with MockitoSu
         await(underTest.fetchApiDocumentationResource(serviceName, "4.0", "resource")(hc))
       }
     }
+
+    "filter the resource if the request is from a person who is not allowed to look at it" in new Setup { //TODO - edit me
+
```
Comment (peteslater-ee) [inline test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala]: Huh?

File: test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala [hunk 2]
```diff
@@ -256,5 +256,13 @@ class DocumentationServiceSpec extends UnitSpec with ScalaFutures with MockitoSu
...
+    }
+
+    "give all of the information to the person if they should see it" in new Setup { //TODO - edit me
+
```
Comment (peteslater-ee) [inline test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala]: Whah?

File: test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala (hunk 1 above)
Comment (rose-garratt) [reply test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala]: Changed!

File: test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala (hunk 2 above)
Comment (rose-garratt) [reply test/unit/uk/gov/hmrc/apidocumentation/services/DocumentationServiceSpec.scala]: Done!

**Pull request merged**